compiler/rustc_feature/src/unstable.rs

```diff
@@ -630,6 +630,8 @@ declare_features! (
     (unstable, strict_provenance_lints, "1.61.0", Some(130351)),
     /// Allows string patterns to dereference values to match them.
     (unstable, string_deref_patterns, "1.67.0", Some(87121)),
+    /// Allows `super let` statements.
+    (unstable, super_let, "CURRENT_RUSTC_VERSION", Some(139076)),
```

Comment:
```suggestion
    /// Allows `super let` statements.
    (incomplete, super_let, "CURRENT_RUSTC_VERSION", Some(139076)),
```

As a lang experiment, this has to go in under an experimental feature flag.